docs/world-compat-details/index.md

```diff
@@ -652,27 +657,38 @@ glibc 提供了两个函数，可以被用于注册信号处理函数，这两
 
 #### `fstat` 相关
 
-新世界的内核中，缺少 `fstat` 和 `newfstatat` 系统调用。这两个系统调用可以被用于获取文件的元数据。
+新世界的低于 (不含) 6.11 版本的内核中，缺少 `fstat` 和 `newfstatat` 系统调用。这两个系统调用可以被用于获取文件的元数据。
 其中，`fstat` 可以获取一个打开的文件描述符（file descriptor, fd）所对应的文件的元数据；
 而 `newfstatat` 则既可以获取一个打开的文件描述符所对应的文件的元数据，也可以获取一个路径所对应的文件的元数据。
 在操作对象上，`statx` 与 `newfstatat` 是一致的，但是可以按需返回更多的信息。因此，在功能上，
-`statx` 是 `fstat` 和 `newfstatat` 的超集，并取代了这两个系统调用。
+`statx` 是 `fstat` 和 `newfstatat` 的超集，并能够实现这两个系统调用。
+然而，Linus 强烈反对在 64 位平台这样做，因此在 6.11 版本内核中重新引入了
+`fstat` 和 `newfstatat` 系统调用。
 
 在 2.38 的 glibc 中，所有 `*stat*` 函数会在编译期通过宏指令检查内核是否提供了 `fstat` 或 `newfstatat` 的定义，
 如果没有，那么这些函数会调用 `statx`[^7] 并负责转换数据结构。这意味着，与旧世界相比，这些 `*stat*` 函数对外呈现的行为是不变的，
 新旧世界的函数是二进制兼容的。本节会着重讨论其向内核发出系统调用的行为区别。
 
+在 2.41 的 glibc 中，如果其在构建时被配置为仅兼容 6.11 或更新版本的内核
```

Comment:
NOT to list backport versions here.  We cannot support them rationally in Glibc.